lib/src/draw_config_context.dart

```diff
@@ -11,14 +11,14 @@ import 'package:path/path.dart' as path;
 import 'exceptions.dart';
 
 const String kAccessToken = 'access_token';
-const String kAuthorizeUri = 'authorize_uri';
+const String kAuthorizeUrl = 'authorize_uri';
```

Comment:
Hm, I'm not sure if this is the best name since it's not actually a URL, just a key. Don't worry about it now, just leave a TODO saying we should think about changing it.